docs/metrics.md

```diff
@@ -2,6 +2,6 @@
 
 Virtual Machine Import uses Prometheus for metrics reporting. The metrics can be used for real-time monitoring. Virtual Machine Import does not persist its metrics, if a member restarts, the metrics will be reset.
 
-| Name               | Description                                                     | Type    | Labels                       |
-|--------------------|-----------------------------------------------------------------|---------|------------------------------|
-| kubevirt_vmimport  | The total number of successfull/failed Virtual Machine imports. | Counter | result=<successful>/<failed> |
+| Name               | Description                                                               | Type    | Labels                                   |
+|--------------------|---------------------------------------------------------------------------|---------|------------------------------------------|
+| kubevirt_vmimport  | The total number of successfull/failed/cancelled Virtual Machine imports. | Counter | result=<successful>/<failed>/<cancelled> |
```

Comment:
when [previewed](https://github.com/kubevirt/vm-import-operator/blob/1c6a2747fcc5c91f50e5426cabf43ceb943560ac/docs/metrics.md) , the content of the 'Labels' column is trimmed. Can you fix it ?